### PR TITLE
Add pill to header Governance item when a vote is in progress

### PR DESCRIPTION
**Pull request by Macxim**
Screen Shot 2022-05-09 at 12 52 24 PM

### CODE REVIEW

File: web/components/header.tsx [hunk 1]
```diff
@@ -28,6 +32,49 @@ export const Header: React.FC<HeaderProps> = ({ signOut, setShowSidebar }) => {
...
   const name = bnsName();
+  const contractAddress = process.env.REACT_APP_CONTRACT_ADDRESS || '';
+  const stxAddress = useSTXAddress();
+  const [isVotingOpen, setisVotingOpen] = useState(false);
+
+  useEffect(() => {
+    let mounted = true;
+
+    const getData = async () => {
+      const proposals = await callReadOnlyFunction({
```
Comment (philipdesmedt) [inline web/components/header.tsx:43]: REQ: we don't need to fetch v2 proposals since there will never be an open one there

**Review by philipdesmedt — CHANGES_REQUESTED** [review]

File: web/components/header.tsx (hunk 1 above)
Comment (Macxim) [reply web/components/header.tsx:43]: Yeah. I wasn't sure. I'll change this. 👍

File: web/components/header.tsx [hunk 2]
```diff
@@ -28,6 +32,37 @@ export const Header: React.FC<HeaderProps> = ({ signOut, setShowSidebar }) => {
...
+        contractAddress,
+        contractName: 'arkadiko-governance-v3-1',
+        functionName: 'get-proposals',
+        functionArgs: [],
+        senderAddress: stxAddress || '',
```
Comment (philipdesmedt) [inline web/components/header.tsx:48]: REQ: this page will fail when the user is not logged in (e.g. try incognito)

The fix is to remove this `stxAddress` completely (also line 36 + the import) and just use `contractAddress` as the sender address

**Review by philipdesmedt — CHANGES_REQUESTED** [review]

File: web/components/header.tsx (hunk 2 above)
Comment (Macxim) [reply web/components/header.tsx:48]: Right, good catch. Will update.

**Pull request merged**